Update DRM design documentation

aucuparia opened the pull request.

aucuparia committed:
Update DRM design documentation

aucuparia committed:
Rename project manager in DRM workflow sequence diagram

aucuparia committed:
Add OSF preregistration to DRM workflow sequence diagram

aucuparia committed:
Remove obsolete DRM installation instructions

stsnel commented on docs/administration/installing-datarequest-module.md:
```diff
@@ -56,8 +45,6 @@ Instructions:
     irule -r irods_rule_engine_plugin-irods_rule_language-instance 'uuGroupAdd("datarequests-research-datamanagers", "datarequest", "research", "Datamanagers", "", *status, *message);' null ruleExecOut
```
I think it would be good to also add separate commands for Yoda 1.9 here, since `uuGroupAdd` has additional required parameters as of Yoda 1.9 (CO identifier, metadata schema, expiration date).

The 1.9 uuGroupAdd commands would look like this (the difference is the extra empty parameters):
```
irule -r irods_rule_engine_plugin-irods_rule_language-instance 'uuGroupAdd("datarequests-research-datamanagers", "datarequest", "research", "", "", "Datamanagers", "", "", *status, *message);' null ruleExecOut
```

aucuparia replied:
Done!

stsnel reviewed (commented):
This looks great in general :+1: 

I have one specific comment (see above), and one question:

The `Technical description` section mentions that `multiple research groups may use a Yoda instance, multiple data manager and DAC groups may be created`. Is this sentence, along with the `brainmorph` example group names still accurate, considering the hardcoded group names (`GROUP_DM`, `GROUP_DAC`, `GROUP_PM`) in the ruleset?

aucuparia committed:
Reword DRM group naming in documentation

DRM user groups are currently not arbitrarily named. This commit replaces the instances in the DRM documentation where this is assumed with the fixed group names currently in use by the DRM.

aucuparia committed:
Add post-Yoda 1.9 irule invocation to DRM installation documentation

stsnel reviewed (approved):
Looks good to me :+1:
